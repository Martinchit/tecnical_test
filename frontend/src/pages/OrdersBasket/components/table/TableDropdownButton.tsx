import styled from 'styled-components';
import { Dropdown } from 'react-bootstrap';

export const TableDropdownButton = styled(Dropdown.Toggle)`
  width: 100%;
  border: 1px solid #b5b5b5;
  display: flex;
  justify-content: space-between;
  align-items: center;
  min-height: 40px;
}
`;
