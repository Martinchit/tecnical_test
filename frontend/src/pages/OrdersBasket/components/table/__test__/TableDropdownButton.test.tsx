import React from 'react';
import { shallow } from 'enzyme';

import { TableDropdownButton } from '../TableDropdownButton';

describe('TableDropdownButton', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<TableDropdownButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
