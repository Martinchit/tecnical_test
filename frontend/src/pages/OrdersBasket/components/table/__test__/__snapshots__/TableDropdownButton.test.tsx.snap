// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableDropdownButton renders correctly 1`] = `
<DropdownToggle
  className="sc-AxjAm jSQqsX"
/>
`;
